src/main/java/org/mule/extension/oauth2/internal/AbstractGrantType.java

```diff
@@ -61,9 +63,15 @@
   private static final String REFRESH_TOKEN_EXPRESSION = "#[payload.refresh_token]";
   private static final String EXPIRATION_TIME_EXPRESSION = "#[payload.expires_in]";
 
-  // TODO MULE-11412 Add @Inject
+  @Inject
```

Comment:
what is this still being used for?

Reply:
for initializing the tokenManager. The muleContext is needed for dependency injection